src/terminal/parser/ut_parser/InputEngineTest.cpp

```diff
@@ -521,11 +580,13 @@ void InputEngineTest::RoundTripTest()
         auto inputKey = IInputEvent::Create(irTest);
         terminalInput.HandleKey(inputKey.get());
     }
+
+    // TODO GH #4405: This test fails.
+    // VerifyExpectedInputDrained();

```

Comment:
Can you just comment the whole test out or something? There's no point in running all the rest of the setup and stuff while it's known broken.

That or do the thing where we do 
```
Log::Result(WEX::Logging::TestResults::Skipped);
return;
```
up at the top until the TODO is solved.